frontend/src/metabase/query_builder/components/DataSelector.jsx

```diff
@@ -174,33 +174,61 @@ export default class DataSelector extends Component {
     renderDatabaseSchemaPicker() {
         const { selectedSchema } = this.state;
 
-        let sections = this.state.databases.map(database => {
-            return {
+        // for nested queries we want to present some different styling so we need
+        // a way to identify this "database"
+        // for now we use it's unique ID that the backend specifies
+        // TODO (@kdoh) - have a nice heart to heart with whoever chose this value
+        const savedQuestionId = -1337
```

Comment:
@camsaul generally yes but since we're thinking of having a nicer identifier (that's I'm guessing will be a boolean?) I didn't bother to do that here.